hadoop-plugin/src/main/groovy/com/linkedin/gradle/hadoopdsl/job/HdfsToEspressoJob.groovy

```diff
@@ -0,0 +1,130 @@
+package com.linkedin.gradle.hadoopdsl.job;
+
+import com.linkedin.gradle.hadoopdsl.NamedScope;
+
+/**
+ * Job class for type=hdfsToEspressoJob jobs.
+ *
+ * <p>
+ * These are documented internally at LinkedIn at https://iwww.corp.linkedin.com/wiki/cf/pages/viewpage.action?pageId=145670487
+ * This is the job type to move data from HDFS to Espresso
+ * <p>
+ * In the DSL, a HdfsToEspressoJob can be specified with:
+ * <pre>
+ *   hdfsToEspressoJob('jobName') {
+ *     qps 1000                        //Required
+ *     sourceHdfsPath '/job/data/src'  //Required
+ *     espressoEndpoint 'www.foo.bar'  //Required
+ *     espressoDatabaseName 'Account'  //Required
+ *     espressoTableName 'Person'      //Required
+ *     espressoContentType 'JSON/TEXT' //Required
+ *     espressoKey 'id'                //Required
+ *     espressoSubkeys 'sub_id,name'   //Optional
+ *     espressoOperation 'put'         //Required
+ *     errorHdfsPath '/job/data/err'   //Required
+ *     set properties: [
+ *       'force.error.overwrite' : true,
+ *       'propertyName1' : 'propertyValue1',
+ *       'propertyName2' : 'propertyValue2',
+ *     ]
+ *   }
+ * </pre>
+ */
+class HdfsToEspressoJob extends Job {
+  int qps;
+
+  String sourceHdfsPath;
+
```

Comment:
Will remove the white space. Actually, I divided parameters into three categories, general config, source, and target and that's where these whitespaces are coming from. If you like alphabetical order, I am good with that as well.